### ISSUE TITLE
Dashboard for logged in user

### ISSUE THREAD

**Issue by lcabrini** [lcabrini/sokms#3]
A user that has just logged in should be taken to a dashboard.

**Comment by lcabrini** [comment]
There is a dashboard already, provided by auth. We will have to customize it to suit us however.

**Comment by CollinsLord** [comment]
Am able to split the dashboard into two, pushing it for you to take a look.

**Comment by CollinsLord** [comment]
Please is that okay.

**Comment by lcabrini** [comment]
Am having a problem with missing HtmlServiceProvider. Tried composer install, but not working.

**Comment by lcabrini** [comment]
Are there any dependencies added? If so, they must be added to composer.

**Comment by CollinsLord** [comment]
No everything is working here perfectly.

**Comment by lcabrini** [comment]
I see. I have this:

    % php artisan serve                                             
    
                                                               
      [Symfony\Component\Debug\Exception\FatalThrowableError]  
      Class 'Collective\Html\HtmlServiceProvider' not found

**Comment by lcabrini** [comment]
Maybe it is due to this: https://stackoverflow.com/questions/28541051/class-illuminate-html-htmlserviceprovider-not-found-laravel-5

**Comment by CollinsLord** [comment]
Do you try the Composer update

**Comment by lcabrini** [comment]
I did composer install instead. Right now trying composer update.

**Comment by CollinsLord** [comment]
okay

**Comment by lcabrini** [comment]
Issue with composer update:

    % composer update                                               [git:master:U:]
    Loading composer repositories with package information
    Updating dependencies (including require-dev)
    Nothing to install or update
    Generating optimized autoload files
    > Illuminate\Foundation\ComposerScripts::postUpdate
    > php artisan optimize
    
                                                               
      [Symfony\Component\Debug\Exception\FatalThrowableError]  
      Class 'Collective\Html\HtmlServiceProvider' not found    
                                                           

    Script php artisan optimize handling the post-update-cmd event returned with error code 1

**Comment by CollinsLord** [comment]
please check if your .evn file is not mess up???

**Comment by lcabrini** [comment]
I'll do that as well, but first I noticed something in the page I linked above. It seems the composer updated needs to be done before the lines are added to config/app.php. Am going to try to comment out the relevant lines, do a composer update and then uncomment.

**Comment by CollinsLord** [comment]
okay i will be waiting for you.
@Prince-linux  i hope you not facing the same problem

**Comment by lcabrini** [comment]
Still not working. As soon as I uncomment any of the lines

    // Collective\Html\HtmlServiceProvider::class,
    // Unisharp\Ckeditor\ServiceProvider::class,

I get an error. It doesn't seem like either of these is pulled in via composer. I cannot find this line in composer.json:

    "laravelcollective/html": "~5.0"

**Comment by lcabrini** [comment]
Could you check your composer file? Maybe you added something that didn't get committed and/or pushed?

**Comment by CollinsLord** [comment]
okay

**Comment by CollinsLord** [comment]
Am sending you the SOK file via gmail

**Comment by CollinsLord** [comment]
via Whatsapp sorry

**Comment by lcabrini** [comment]
Which SOK file, by the way? Is it related to the dashboard issue?

**Comment by CollinsLord** [comment]
if you can compare yours with the one i send to you via whatsapp....

**Comment by lcabrini** [comment]
Yeah, I just did. Can't see any difference. Plus, there is no collectivehtml folder in vendor. Could you have it some other place?

**Comment by lcabrini** [comment]
OK. I fixed it. Pushing updated composer.json.

**Comment by CollinsLord** [comment]
okay.

**Comment by lcabrini** [comment]
Dealt with in #21 .

**Comment by CollinsLord** [comment]
Looking at the dashboard is it okay

**Comment by lcabrini** [comment]
Please have a look at #22 . This is preventing me from looking at the dashboard. :)

**Comment by Prince-linux** [comment]
I have to take a look too .. @CollinsLord ..
